### DISCUSSION TITLE
How can I pass CSS class to livewire component ?

### DISCUSSION THREAD

**Discussion by t-prod** [livewire/livewire#7942] [1. Help]
Hi,

I have several instances of a component in different pages. I would like to skin it in checkout by adding a wrapping class

On checkout : 

`<livewire:iban-form class="bg-light" />`

On other pages 

`<livewire:iban-form />`

Can I use blade component directives ? Do you have some examples ?

Thanks for your help.

**Comment by shubdragon** [comment]
you could try this way:

`<livewire:iban-form :classes="'bg-light text-dark etc'" />`
**or**
 `@livewire('iban-form', ['classes' => 'bg-light text-dark etc'])`
**or**
 `@livewire('iban-form', ['classes' => $classStringsFromDynamicHeaven])`

and inside iban-form component class make a public variable in same name `public ?string $classes;`

and inside the component blade `<div class="existing class {{ $classes }}"> </div>`

This is the one way if anyone have better way you can check

**Comment by t-prod** [comment]
Thanks for your answer I thought we could do the same technique as a blade component and props.
You're right, for now is the only solution.
